crates/runtime/src/accelerated_table/refresh.rs

```diff
@@ -454,6 +455,7 @@ impl Refresher {
             initial_load_completed: Arc::new(AtomicBool::new(false)),
             disable_federation: false,
             semaphore: None,
+            notifier: None,
```

Comment:
Not immediately obvious this notifies on completion. Either a comment or rename the variable to `on_completion_nofication` or similar.